src/php/fpm/conf/zz-docker.conf.template

```diff
@@ -0,0 +1,81 @@
+[global]
```

Comment:
git detected this as a delete/addition unfortunately. i can make git detect it as a move if using 2 commits, but that's against your contributing guidelines.

Reply:
In this case I think it's fine to make in 2 commits, it'll help on the review as well! Thanks

Reply:
ok well, i did that, and if you view the 2 commits individually it looks fine.
but the diff of the whole PR shows a delete/addition.
strange,